src/UI/Art/SpriteArtist/SpriteMapView.java

```diff
@@ -324,12 +324,12 @@ private BufferedImage renderMap()
     for( DamagePopup popup : mapController.getDamagePopups() )
       drawDamagePreview(mapGraphics, popup, gameMap.isLocationEmpty(popup.coords));
 
-    // When we draw the map, we want to center it if it's smaller than the view dimensions
+    // When we draw the map, we want to right-justify it if it's smaller than the view dimensions
```

Comment:
Why? This looks pretty weird at full screen.

Reply:
I mean yeah, it brings up the extra CO overlays sooner, but still. Not sure it's a net improvement.

Reply:
Hmm. I suppose I could right-justify until there's "plenty of space" and then go as close to center-aligned as possible?